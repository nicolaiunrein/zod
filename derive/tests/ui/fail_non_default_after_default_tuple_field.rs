mod test {
    struct Ns;
    impl zod_core::Namespace for Ns {
        const NAME: &'static str = "MyNs";
    }

    #[derive(zod_derive::Zod, serde::Serialize)]
    #[zod(namespace = "Ns")]
    struct Struct(#[serde(default)] u8, bool);
}

fn main() {}
