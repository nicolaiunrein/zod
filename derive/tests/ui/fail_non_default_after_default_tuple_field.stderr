error: zod: `non-default field follows default field`
 --> tests/ui/fail_non_default_after_default_tuple_field.rs:9:41
  |
9 |     struct Struct(#[serde(default)] u8, bool);
  |                                         ^^^^
